lib/checkstl.cpp

```diff
@@ -741,6 +741,11 @@ static ValueFlow::Value getLifetimeIteratorValue(const Token* tok, MathLib::bigi
     }
     if (values.size() == 1)
         return values.front();
+    it = std::find_if(values.cbegin(), values.cend(), [](const ValueFlow::Value& v) {
+        return v.tokvalue->str() == ".";
```

Comment:
Stupid question.. Are you sure that ValueFlow::getLifetimeObjValues only return values that have tokvalue?

Reply:
If you are sure then feel free to merge.

Reply:
We always set the `tokvalue` together with `ValueType::LIFETIME`. Although this should probably be explicitly documented somewhere...